examples/stable_diffusion_v2/T2I-Adapter.md

```diff
@@ -19,34 +19,42 @@ to SD models, making them easy to integrate and use. The overall architecture of
 - **Simple and lightweight**: 77M parameters for full and 5M parameters for light adapters.
 - **Composable**: Several adapters can be combined to achieve multi-condition control.
-- **Generalizable**: Can be directly used on custom models as long as they fine-tuned from the same model (e.g., use
+- **Generalizable**: Can be directly used on custom models as long as they are fine-tuned from the same model (e.g., use
   T2I-Adapters trained on SD 1.4 with SD 1.5 or Anything anime model).
 
 ## Pretrained Models
 
 <div align="center">
 
-| Tasks        | SD version | Weights                                                                                                          |
-|--------------|------------|------------------------------------------------------------------------------------------------------------------|
-| Canny        | 1.5        | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_canny_sd15v2-c484cd69.ckpt)    |
-| Color        | 1.4        | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_color_sd14v1-7cb31ebd.ckpt)    |
-| Depth        | 1.5        | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_depth_sd15v2-dc86209b.ckpt)    |
-| KeyPose      | 1.4        | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_keypose_sd14v1-ee27ccf0.ckpt)  |
-| OpenPose     | 1.4        | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_openpose_sd14v1-ebcdb5cb.ckpt) |
-| Segmentation | 1.4        | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_seg_sd14v1-1d2e8478.ckpt)      |
-| Sketch       | 1.5        | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_sketch_sd15v2-6c537e26.ckpt)   |
-| Style        | 1.4        | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_style_sd14v1-a620ae97.ckpt)    |
+| SD Compatibility | Task         | SD Train Version | Dataset                                | FID ↓ | CLIP Score ↑ | Context         | Train Time      | Throughput   | Recipe                                      | Weights                                                                                                          |
+|:----------------:|--------------|:----------------:|----------------------------------------|-------|--------------|-----------------|-----------------|--------------|---------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+|       2.x        | Segmentation |       2.1        | [COCO-Stuff](#segmentation-coco-stuff) | 26.10 | 26.32        | D910Bx4-MS2.1-G | 10h 35m / epoch | 39.2 img / s | [yaml](configs/t2i-adapter/v2.1_train.yaml) | [Download](https://download.mindspore.cn/toolkits/mindone/t2i-adapters/t2iadapter_seg_sd21-86d4e0db.ckpt)        |
```

Comment:
It would be more clear to put our training and evaluation results separately in another table, e.g. in Evaluation section.

And leave the pretrained models table only to keep the previously released models.

Reply:
Done